daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -322,8 +312,10 @@ final case class Compiler(packages: PackageId PartialFunction Package) {
               case BTextIntercalate => SBTextIntercalate
 
               // Implemented using normal SExpr
-              case BFoldl | BFoldr | BEqualList | BCoerceContractId | BTextMapEmpty |
-                  BGenMapEmpty =>
+              case BFoldl | BFoldr | BCoerceContractId | //
```

Comment:
Why the `//`s?

Reply:
trying to fight against auto-formatter.

dropped